etl/importer.py

```diff
@@ -207,6 +208,10 @@ def import_psts(
     if clean:
         logger.warning(f"Deleting {account.title} account files (if exists)")
         account.files.all().delete()
+    if clean_file:
```

Comment:
It'd be nice to have a test that removes the file, but I realize we don't have anything testing this portion of the code yet. Optional.

Reply:
I'll create a ticket for testing the importer.